x-pack/plugin/transform/qa/build.gradle

```diff
@@ -10,5 +10,9 @@ gradle.projectsEvaluated {
     if (dependenciesInfo) {
       dependenciesInfo.enabled = false
     }
+    Task dependenciesGraph = project.tasks.findByName('dependenciesGraph')
```

Comment:
This seem to duplicate the logic in the root build.gradle file we have for disabling the task